src/main/java/com/mongodb/jdbc/MongoDriver.java

```diff
@@ -86,6 +87,8 @@
             MAJOR_VERSION = 0;
             MINOR_VERSION = 0;
         }
+        String name = unit.getClass().getPackage().getImplementationTitle();
+        NAME = (name != null) ? name : "mongodb-jdbc";
```

Comment:
Under what circumstances would we expect `name` to be `null`?

Reply:
I found when running the gradle test during `gradle clean build`, the values returned for `getImplementationTitle()` and `getImplementationVersion()` are `null`.  

It returns the expected version and name values when running a program that uses the `mongodb-jdbc-1.0.4-SNAPSHOT.jar` JAR file.  I think it has something to do with the `META-INF\MANIFEST.MF` file which exists in the JAR.